File: src/test/java/at/chaoticbits/render/HtmlServiceTest.java
package at.chaoticbits.render;

import at.chaoticbits.coinmarket.TestData;
import org.testng.Assert;
import org.testng.annotations.Test;

import java.io.InputStream;

public class HtmlServiceTest {


    @Test
    public void testGenerateCryptoDetailsImage() {
        InputStream image = HtmlImageService.getInstance().generateCryptoDetailsImage(TestData.currencyDetails());
        Assert.assertNotNull(image);
    }


}

File: src/test/java/at/chaoticbits/render/HtmlServiceTest.kt
package at.chaoticbits.render

import at.chaoticbits.coinmarket.TestData
import org.testng.Assert
import org.testng.annotations.Test

import java.io.InputStream

class HtmlServiceTest {


    @Test
    fun testGenerateCryptoDetailsImage() {
        val image = HtmlImageService.generateCryptoDetailsImage(TestData.currencyDetails())
        Assert.assertNotNull(image)
    }


}
